front_end/bindings/BlackboxManager.js

```diff
@@ -341,7 +341,63 @@ Bindings.BlackboxManager = class {
     if (!script.sourceURL && !script.sourceMapURL)
       return Promise.resolve();
     const blackboxed = this._isBlackboxedScript(script);
-    return this._setScriptState(script, blackboxed ? [{lineNumber: 0, columnNumber: 0}] : []);
+    var ranges =  blackboxed ? [{lineNumber: 0, columnNumber: 0}] : [];
+    if (script.sourceURL.includes('dart_sdk.js')) {
```

Comment:
nit: can you be a bit more strict and make this endsWith '/dart_sdk.js?'

Reply:
That requires parsing the URL so we can get just the path part. Did it by doing this check first, and then the more strict check inside, since this gets called for every file.